<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(71, 71, 71);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="a_header">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>74</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(62, 62, 62);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame">
        <property name="maximumSize">
         <size>
          <width>74</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 154, 87);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="styleSheet">
            <string notr="true">border: none;</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="../recursos.qrc">
             <normaloff>:/iconos/icons/file.svg</normaloff>:/iconos/icons/file.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>38</width>
             <height>38</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">color: white;
font: 27px &quot;Bahnschrift Condensed&quot;;</string>
           </property>
           <property name="text">
            <string>Conversión a través de un .txt</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="b_content">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>385</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QListWidget" name="view_temp">
        <property name="styleSheet">
         <string notr="true">border: none;
color: white;
padding: 9px;
font: 14pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(47, 47, 47);
border-radius: 12px;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_result">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>68</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_save">
        <property name="styleSheet">
         <string notr="true">QPushButton {
border: none;
color: white;
font: 11pt &quot;Cascadia Code&quot;;
background-color: rgb(255, 148, 121);
border-top-right-radius: 9px;
border-bottom-left-radius: 9px;
}

QPushButton:pressed {
border: 2px solid rgb(255, 148, 121);
background-color: none;
color: rgb(255, 148, 121);

}</string>
        </property>
        <property name="text">
         <string>Guardar resultados</string>
        </property>
        <property name="icon">
         <iconset resource="../recursos.qrc">
          <normaloff>:/iconos/icons/check.svg</normaloff>:/iconos/icons/check.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>27</width>
          <height>27</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../recursos.qrc"/>
 </resources>
 <connections/>
</ui>
